mautic/mautic
# Robots.txt dynamic and embeddable

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | Y
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
This PR create Robots.txt embeddable similar like BuildJSSubscriber (mtc.js). 
Robots.txt can be now dynamic and extented by plugin and developers.
Also some inspirations https://www.link-assistant.com/blog/10-robots-txt-files-worth-to-have-a-look-at/

#### Steps to test this PR:
1. Open http://yourmautic.tlc/robots.txt and backup it
1.  Apply PR and remove robots.txt from root directory
2.  Open http://yourmautic.tlc/robots.txt  and see if works and contain same content like previous static robots.txt file
